### PR TITLE
🌐 Add Portuguese translation for Tutorial - Path Parameters

### PR DESCRIPTION
**Pull request by jessicapaz**
This PR translates the tutorial/path-params.md and add this page to the mkdocs.yml file.

### CODE REVIEW

File: docs/pt/docs/tutorial/path-params.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
+# Parâmetros de rota
+
+Você pode declarar "parâmetros" ou "variáveis" de rota com a mesma sintaxe usada para formatar strings em Python:
```
Comment (Kludex) [inline docs/pt/docs/tutorial/path-params.md:3]: Is there a reason for `parametros` and `variaveis` being wrapped by the double quotes?

File: docs/pt/docs/tutorial/path-params.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+## <abbr title="também conhecido como: serialização, parseamento, marshalling">Conversão</abbr> de Dados
+
+Se você executar este exemplo e abrir o seu navegador em <a href="http://127.0.0.1:8000/items/3" class="external-link" target="_blank">http://127.0.0.1:8000/items/3</a>, você verá a seguinte resposta:
+
+```JSON
+{"item_id":3}
+```
+
+!!! check
+    Observe que o valor que sua função recebeu (e retornou) é `3`, como o tipo Python `int`, não uma string `"3"`.
```
Comment (Kludex) [inline docs/pt/docs/tutorial/path-params.md:40]:
```suggestion
    Observe que o valor que sua função recebeu (e devolveu) é `3`, como o tipo Python `int`, não uma string `"3"`.
```

File: docs/pt/docs/tutorial/path-params.md (hunk 1 above)
Comment (jessicapaz) [reply docs/pt/docs/tutorial/path-params.md:3]: In English it's the same. https://fastapi.tiangolo.com/tutorial/path-params/#path-parameters

Comment (Kludex) [reply docs/pt/docs/tutorial/path-params.md:3]: Ah... ok.

File: docs/pt/docs/tutorial/path-params.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+Se você executar este exemplo e abrir o seu navegador em <a href="http://127.0.0.1:8000/items/3" class="external-link" target="_blank">http://127.0.0.1:8000/items/3</a>, você verá a seguinte resposta:
+
+```JSON
+{"item_id":3}
+```
+
+!!! check
+   Observe que o valor que sua função recebeu (e devolveu) é `3`, como o tipo Python `int`, não uma string `"3"`.
+
+    Então, com essa declaração de tipo, **FastAPI** oferece a você uma <abbr title="convertendo a string que vem de uma requisição HTTP em dados Python">"conversão"</abbr> automática de requisição.
```
Comment (lsglucas) [inline docs/pt/docs/tutorial/path-params.md:42]: This check is not showing correctly on the preview website. 

I believe the following suggestion will fix it.
```suggestion
!!! check
    Observe que o valor que sua função recebeu (e devolveu) é `3`, como o tipo Python `int`, não uma string `"3"`.

    Então, com essa declaração de tipo, **FastAPI** oferece a você uma <abbr title="convertendo a string que vem de uma requisição HTTP em dados Python">"conversão"</abbr> automática de requisição.
```

**Review by stlucasgarcia — CHANGES_REQUESTED** [review]
There are two markdown typos to fix, the translation is perfect 👍🏻

**Comment by cassiobotaro** [comment]
@jessicapaz Can you take a look at issues reported by @lsglucas?

**Comment by LorhanSohaky** [comment]
Can we close it?

Relateds #3664 #4722

**Comment by tiangolo** [comment]
Thanks all! As this was handled in another PR I'm gonna close this one. ☕

**Pull request closed without merging**